gui/default/index.html

```diff
@@ -257,6 +257,33 @@ <h3 class="panel-title">
       </div>
     </div>
 
+    <!-- Panel: FS watcher errors -->
+
+    <div ng-if="sizeOf(fsWatcherErrorMap()) > 0" class="row">
```

Comment:
you can assign the return value of that on the div temporarily I think, so there wouldn't be 2 calls to this.

Reply:
It looks like that doesn't work because ng-init is only evaluated once on startup, then never again.